### ISSUE TITLE
Allow Transcribers to check that the Church has been set up.

### ISSUE THREAD

**Issue by edickens** [FreeUKGen/MyopicVicar#803]

**Comment by edickens** [comment]
On 15 May a file had all records rejected because the Church did not match...

Started on the file CONLEZBU.CSV for Mellerstain at 2016-05-15 01:42:15 +0100. <p>
Code set 
Checking for undefined with 
The church name St Briocus is not in the database for Lezant at line 1. <br>

In this case the Church name does not (quite correctly) match...

Lezant 	St Michael (St. Briochus)

I have taken this from the "Transcriptions" link on the homepage, but if this had been the first set of transcriptions for this Church, then this information is not available to the transcriber.
The SC should tell the transcriber what to enter, but it is easier for the transcriber to copy and paste the exact spelling if they had access to the "All places" Action for all Counties, but to View only.
If we also match Register Type in the future, then this is even more necessary.   E

PS.  What is "Checking for undefined with" which is shown for every file.

**Comment by edickens** [comment]
A transcriber has no way of knowing the correct spelling of the Church, as set up, that they are about to transcribe.  Access to the list of "All Places" is not available to them.  Until the first entry is correctly uploaded, the Church does not appear in "Transcriptions".  The transcriber relies on the SC to tell them the correct spelling of the Church name.  We have had problems this last week in this area.

**Comment by SteveBiggs** [comment]
That's true, but the transcriber also relies on the CC/SC to have set up the Church on FR before they can upload a file and to tell them the file code to use.  I think we have to assume that the SC will provide all this information directly to the transcriber.

**Comment by edickens** [comment]
In the new FR Assignment system it will be in the confirmation email, so not needed now.

**Comment by SteveBiggs** [comment]
@edickens what's not needed now?

As you know, most SC's do not use the image server, so they will continue sending info emails manually even when the new system goes live.

**Comment by edickens** [comment]
It is not necessary for transcribers to be able to see the list of Places/Church Names before any records have been uploaded.  After one record has been uploaded, it all appears in "Transcriptions".

Yes.  Hopefully they will see that it is better to use the server.
